packages/babel-plugin-transform-runtime/package.json

```diff
@@ -14,6 +14,7 @@
   "dependencies": {
     "@babel/helper-module-imports": "^7.0.0",
     "@babel/helper-plugin-utils": "^7.0.0",
+    "lodash": "^4.17.11",
```

Comment:
Can you add a dependency on `lodash.merge` instead?

Reply:
Other packages depend on lodash, so rather than introducing a new dependency I figured I'd just depend on the same and allow it to be deduped.

Also I believe the lodash maintainers no longer recommend using the modularized packages.